pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountThetaSketchAggregationFunction.java

```diff
@@ -576,6 +616,69 @@ private SetOperationBuilder getSetOperationBuilder() {
         : SetOperation.builder().setNominalEntries(_thetaSketchParams.getNominalEntries());
   }
 
+  /**
+   * Validates that the function context's substitution parameters ($1, $2, etc) does not exceed the number
+   * of predicates passed into the post-aggregation function.
+   *
```

Comment:
Also validate that there are 2 arguments for `DIFF`

Reply:
Done